### PR TITLE
(SIMP-7487) Add EL8 support

### PR DESCRIPTION
**Pull request by lnemsick-simp**
- Via updated acceptance tests, verified that the existing
  configuration worked on EL8 and that EL7 and EL8 interoperate.
- Update the upper bound of simp-simplib to < 5.0.0
- Add REFERENCE.md

SIMP-7487 #close
SIMP-7488 #close
SIMP-7436 #close
SIMP-7383 #close

### CODE REVIEW

**Comment by lnemsick-simp** [comment]
Acceptance tests run on a local GitLab

File: spec/acceptance/suites/default/10_server_client_spec.rb [hunk 1]
```diff
@@ -8,6 +8,7 @@
       skip('You need at least two hosts in your nodeset to run this test')
     end
   else
+    # test interoperability between 1st and 2nd server in the node set
     server1 = hosts[0]
```
Comment (trevor-vaughan) [inline spec/acceptance/suites/default/10_server_client_spec.rb]: This needs to loop through all hosts and test the interconnections between each pair to make sure all the connections function properly.

File: spec/acceptance/suites/default/20_server_client_stunnel_spec.rb [hunk 2]
```diff
@@ -8,6 +8,7 @@
       skip('You need at least two hosts in your nodeset to run this test')
     end
   else
+    # test interoperability between 1st and 2nd server in the node set
     server1 = hosts[0]
```
Comment (trevor-vaughan) [inline spec/acceptance/suites/default/20_server_client_stunnel_spec.rb]: This also needs to loop through all combinations.

**Comment by trevor-vaughan** [comment]
This has an example of the pairing loops https://github.com/simp/pupmod-simp-stunnel/blob/master/spec/acceptance/suites/default/20_connectivity_spec.rb#L18

**Comment by lnemsick-simp** [comment]
✔️ Acceptance tests run on a local GitLab

**Pull request merged**